amplitude/types/event.go

```diff
@@ -13,6 +17,9 @@ type Event struct {
 
 	// DeviceID is a device-specific identifier. The value is ignored if EventOptions.DeviceID is set.
 	DeviceID string `json:"-"`
+
+	RetryAt    time.Time `json:"-"`
+	RetryCount int       `json:"-"`
```

Comment:
I would like to keep these values off the Event struct. Instead this could be managed internally by the response processor.

Reply:
Fixed